catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.email.xml

```diff
@@ -21,15 +21,15 @@
 
         <AD description="Set the subject line template."
             name="Subject" id="subjectTemplate" required="true" type="String"
-            default="[%[workspaceId]] workspace query notice"/>
+            default="[%[attribute=id]] workspace query notice"/>
 
         <AD description="Set the body template."
             name="Body" id="bodyTemplate" required="true" type="String"
-            default="The workspace '%[workspaceId]' contains up to %[hitCount] query hits."/>
+            default="The workspace '%[attribute=id]' contains up to %[hitCount] query hits."/>
 
         <AD description="Set the hostname of the mail server."
             name="Mail Server" id="mailHost" required="true" type="String"
-            default="connexta-com.mail.protection.outlook.com"/>
+            default="example.mail.protection.outlook.com"/>
```

Comment:
Is there a reasonable default value for the mail host? I'm thinking it should be an empty string.